.github/ionic-issue-bot.yml

```diff
@@ -26,6 +26,15 @@ closeAndLock:
 
 comment:
   labels:
+    - label: 'holiday triage'
+      message: >
+        Thanks for the issue! This issue has been labeled as `holiday triage`. With the winter holidays quickly approaching, much of the Stencil Team will soon be taking time off. During this time, issue triaging and PR review will be delayed until the team begins to return. After this period, we will work to ensure that all new issues & PRs are properly triaged.
+
```

Comment:
The double newline is needed, otherwise, we get:
```
        Thanks for the issue! This issue has been labeled as `holiday triage`. With the winter holidays quickly approaching, much of the Stencil Team will soon be taking time off. During this time, issue triaging and PR review will be delayed until the team begins to return. After this period, we will work to ensure that all new issues & PRs are properly triaged.
        In the meantime, please read our [Winter Holiday Triage Guide](https://github.com/ionic-team/stencil/issues/3922) for information on how to ensure that your issue is triaged correctly.
        Thank you!
```